en/get-started.md

```diff
@@ -4,3 +4,396 @@ summary: Learn how to quickly deploy a TiDB cluster on Kubernetes using TiDB Ope
+    metadata:
+      name: tidb
+      namespace: db
+    spec:
+      cluster:
+        name: basic
+      replicas: 1
+      template:
+        metadata:
+          annotations:
+            author: pingcap
+        spec:
+          version: v8.1.0
+    ```
+
+    ```shell
+    kubectl apply -f tidb.yaml --server-side
+    ```
+
+    </div>
+    </SimpleTab>
+
+    Method 2: save the preceding YAML files to a local directory and deploy the TiDB cluster with a single command:
+
+    ```shell
+    kubectl apply -f ./<directory> --server-side
+    ```
+
+3. Monitor the status of Pod:
+
+    ```shell
+    watch kubectl get pods -n db
+    ```
+
+    <details>
+    <summary>Expected output</summary>
+
+    ```
+    NAME               READY   STATUS    RESTARTS   AGE
+    pd-pd-68t96d       1/1     Running   0          2m
+    tidb-tidb-coqwpi   1/1     Running   0          2m
+    tikv-tikv-sdoxy4   1/1     Running   0          2m
+    ```
+
+    </details>
+
+    After all component Pods start, each component type (`pd`, `tikv`, and `tidb`) will be in the Running state. You can press <kbd>Ctrl</kbd>+<kbd>C</kbd> to return to the command line and proceed to the next step.
+
+## Step 4: Connect to TiDB
+
+Because TiDB supports the MySQL transport protocol and most of its syntax, you can use the `mysql` command-line tool to connect to TiDB directly. The following steps describe how to connect to the TiDB cluster.
+
+### Install the `mysql` command-line tool
+
+To connect to TiDB, you need to install a MySQL-compatible command-line client on the machine where you are running `kubectl`. You can install MySQL Server, MariaDB Server, or Percona Server MySQL executables, or install them from your operating system's software repository.
+
+### Forward port 4000
+
+To connect to TiDB, you need to forward a port from the local host to the TiDB service on Kubernetes.
+
+First, list services in the `db` namespace:
+
+```shell
+kubectl get svc -n db
+```
+
+<details>
+<summary>Expected output</summary>
+
+```
+NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)               AGE
+pd-pd            ClusterIP   10.96.229.12    <none>        2379/TCP,2380/TCP     3m
+pd-pd-peer       ClusterIP   None            <none>        2379/TCP,2380/TCP     3m
+tidb-tidb        ClusterIP   10.96.174.237   <none>        4000/TCP,10080/TCP    3m
+tidb-tidb-peer   ClusterIP   None            <none>        10080/TCP             3m
+tikv-tikv-peer   ClusterIP   None            <none>        20160/TCP,20180/TCP   3m
+```
+
+</details>
+
+In this example, the TiDB service is `tidb-tidb`.
+
+Then, use the following command to forward a local port to the cluster:
+
+```shell
+kubectl port-forward -n db svc/tidb-tidb 14000:4000 > pf14000.out &
+```
+
+If port `14000` is already in use, you can use a different available port. The command runs in the background and forwards output to the file `pf14000.out`, so you can continue to run commands in the current shell session.
```

Comment:
Port `14000` or `4000`? In the heading Forward port 4000, it's  `4000`.

Reply:
It means forward port 4000 to the local port 14000